.sisyphus/templates/ralph-loop-verify.sh

```
@@ -0,0 +1,211 @@
+if [ "$merge_state" != "CLEAN" ]; then
+  fail "mergeStateStatus=$merge_state (expected CLEAN) — CI checks pending/failing or branch out of date"
+fi
+info "mergeStateStatus=CLEAN"
+
+# ----------------------------------------------------------------------------
+# Rule 5: Zero unresolved review threads
+# Every CR thread must be closed out: fix commit, reasoned rejection, or deferral.
+# Silent dismissal (resolving without a reply) is a contract violation.
+# ----------------------------------------------------------------------------
+unresolved=$(gh api graphql \
+  -f query='query($o:String!,$r:String!,$p:Int!){repository(owner:$o,name:$r){pullRequest(number:$p){reviewThreads(first:100){nodes{isResolved}}}}}' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)